ha_lineairdb.h

```diff
@@ -67,6 +67,29 @@ class LineairDB_share : public Handler_share {
   std::unique_ptr<LineairDB::Database> lineairdb_;
 };
 
+class Mysql_lineairdb_translator
```

Comment:
This class should be in a separated file. just like `translator.h`.

Reply:
Understood